tools/scaffold/scripts/schema-scaffold.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+
+# URL to download Atlas Admin API Spec
+atlas_admin_api_spec="https://raw.githubusercontent.com/mongodb/atlas-sdk-go/main/openapi/atlas-api-transformed.yaml"
```

Comment:
Could we explain somewhere why we are using this file instead of https://mongodb-mms-prod-build-server.s3.amazonaws.com/openapi/d05d140a757e46f9ce25ee52b17ae91335003c32-v2-2023-11-15.json? I want to make sure capture the reasons behind this decision so that we don't forget it

Reply:
keep in mind that the SDK will move from a push to a pull mechanism of the open api spec once the FOAS project is completed so we will need to update this link to point to the FOAS

Reply:
I added details the Spec document with the reasoning behind this decision, as well as a point in the risks section. If we are for seeing a significant maintenance cost we can adjust here.